sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3962,11 +4021,14 @@ SUBROUTINE MISCLN
                 ENDIF
                 STP=(MUCAPE(I,J)/D1000)*(ESRH(I,J)/50.)*&
                         ESHRtmp*MUCINtmp
+                GRID1(I,J) = spval
+                IF(T1D(I,J) < spval) THEN
```

Comment:
remove this if block and add to the next if block, e.g.,
IF (STP>0 .and. STP<spval) THEN